src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $transitionSpeed, color $transitionSpeed;
```

Comment:
I can't find a declaration of these vars. Probably you forgot to add file with them to your commit or to deploy the changes